packages/@uppy/status-bar/src/StatusBar.tsx

```diff
@@ -72,8 +73,7 @@ const defaultOptions = {
  * progress percentage and time remaining.
  */
 export default class StatusBar<M extends Meta, B extends Body> extends UIPlugin<
-  StatusBarOptions &
-    Required<Pick<StatusBarOptions, keyof typeof defaultOptions>>,
+  DefinePluginOpts<StatusBarOptions, keyof typeof defaultOptions>,
```

Comment:
is it possible to put the `keyof typeof` inside `DefinePluginOpts`? We would always use the `defaultOptions` pattern, even if there is only one.

Reply:
Alright it was more a nit not blocking.